assemble/src/main/scripts/create-jshell.sh

```diff
@@ -36,6 +36,26 @@ function addAccumuloAPI() {
   echo
 }
 
+function addClientBuild() {
+cat <<EOF
+  URL clientPropUrl = 
+    AccumuloClient.class.getClassLoader().getResource("accumulo-client.properties");
+  String accumuloProp; AccumuloClient client = null;
+  
+  // Does Accumulo properties exists?
+  if (clientPropUrl != null) {
+  
+    // Build Accumulo Client
+    accumuloProp = clientPropUrl.getFile();
```

Comment:
Not all URLs from the class path will sensibly convert to a meaningful File path. It might be useful to use the URL directly for printing the location where it was loaded from. If our Accumulo client builder code doesn't already have a `from(URL)` method, we should probably add one. There are plenty of URLs that Java can read, especially from the class path, that aren't a file on the local filesystem. For example, a file read from inside a jar on the class path (which might look something like `jar:file:///path/to/client/provided/configurationbundle.jar!/accumulo-client.properties`)

Reply:
I don't believe we have a from(URL) method. Just to make sure I understand, is the from(URL) method a separate implementation or an existing Accumulo function where we can perform a function call to grab the properties file path?

Reply:
If we don't have one, we can implement one.

Reply:
In my recent commit, I believe I found a solution to gather the properties content from the URL and use the existing from methods without creating a new from(URL). Essentially I created an InputStream that accesses the properties file from the given URL and used the properties object method to build the client. I tested out my idea by creating two tests where properties file was placed in a JAR file and a regular file directory. The proposed implementation was able to find and load in the properties file in both tests. Since InputStream supports Auto-Closeable, I included a try and resource statement so that the InputStream automatically closes once it's no longer needed.